sebs/fission/fission.py

```diff
@@ -0,0 +1,373 @@
+import logging
+import os
+import shutil
+import subprocess
+import docker
+import json
+from time import sleep
+from typing import Dict, Tuple, List
+from sebs.faas.storage import PersistentStorage
+from sebs.cache import Cache
+from sebs.config import SeBSConfig
+from sebs.faas.function import Function
+from sebs.faas.system import System
+from sebs.fission.fissionFunction import FissionFunction
+from sebs.benchmark import Benchmark
+from sebs.fission.config import FissionConfig
+from sebs.fission.minio import Minio
+from tools.fission_preparation import (
```

Comment:
I prefer that we remove that - the "preparation" script should remain an optional tool helping some users. SeBS should assume that the platform is already running and working.